tests/containers/awardV2/idv/IdvActivityContainer-test.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * IdvActivityContainer-test.js
+ * Created by Jonathan Hill 6/26/2019
+**/
+
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+
+import { IdvActivityContainer } from 'containers/awardV2/idv/IdvActivityContainer';
+import BaseIdvActivityBar from 'models/v2/awardsV2/BaseIdvActivityBar';
+import { mockRedux, mockActions } from '../mockAward';
+import { mockIdvActivity } from '../../../models/awardsV2/mockAwardApi';
+
+jest.mock('helpers/idvHelper', () => require('../awardV2Helper'));
+
+// mock the child component by replacing it with a function that returns a null element
+// jest.mock('components/awardv2/idv/amounts/AggregatedAwardAmounts.jsx', () => jest.fn(() => null));
+
+describe('IdvActivityContainer', () => {
+    const loadAwards = jest.fn();
+    const parseAwards = jest.fn();
+    it('should make an API call for the awards on mount', async () => {
+        const container = mount(<IdvActivityContainer {...mockActions} {...mockRedux} />);
+
+        container.instance().loadAwards = loadAwards;
+        container.instance().parseAwards = parseAwards;
+        await container.instance().componentDidMount();
```

Comment:
are we awaiting a promise in this case? I don't think we typically return anything in the lifecycle methods. Just curious to get your thoughts on this given our previous convos.